accounts/salesforce.py

```diff
@@ -0,0 +1,17 @@
+from simple_salesforce import Salesforce
+from django.conf import settings
+
+
+sf=Salesforce(username=settings.SALES_USER,
+              password=settings.SALES_PASSWORD,
+              security_token=settings.SALES_SECURITY_TOKEN)
+
+
+def faculty_status(accounts_id):
```

Comment:
Yeah - there was a question from Kathi about what would happen if SF was down - how will the app handle that? This should be addressed in the error handling.